ExchangeUnion/xud-docker-bot
# Trigger image builds from bot

### How to test

1. Create a branch in xud-docker
2. Change some images (**remove `.travis.yml` first!**)
3. See if there is a Travis build triggered to build your modified images

### Todos

- [x] Create a PR in xud-docker to remove `.travis.yml`
